src/users/dtos/login-user.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsEmail, IsNotEmpty, IsString } from 'class-validator';
+
+export class LoginDto {
```

Comment:
class명은 파일명이랑 동일하게 가져가면 좋을 것 같아요!
```suggestion
export class LoginUserDto {
```

Reply:
[b207cb3](https://github.com/programmers-Codeine/Pocur_Q_BE/pull/8/commits/b207cb3657840ae4766f4ebedf6d83ad431e69b2)
수정 완료했습니다.